src/integration-test/java/fr/inria/jtravis/helpers/BuildHelperIntegrationTest.java

```diff
@@ -386,6 +401,16 @@ public void testGetLastBuildFromMasterIgnoreNumberNullBuilds() {
         assertNotNull(buildOptional.get().getNumber());
     }
 
+    @Test
+    @Category(IntegrationTest.class)
+    public void testGetLastBuildFromRenamedRepository() {
+        String oldSlug = "alibaba/dubbo";
+        Optional<Build> buildOptional = getJTravis().build().last(oldSlug);
+
+        assertTrue(buildOptional.isPresent());
+        assertNotNull(buildOptional.get().getNumber());
```

Comment:
Add an assert to check the repo getId slug is the new one

Reply:
Done.